pkg/analysis/passes/brokenlinks/brokenlinks.go

```diff
@@ -149,7 +149,7 @@ func run(pass *analysis.Pass) (interface{}, error) {
 			defer resp.Body.Close()
 
 			if resp.StatusCode != http.StatusOK {
-				if isGitHubURL(url.url) && strings.Contains(resp.Status, "429 Too Many Requests") {
+				if resp.StatusCode == http.StatusTooManyRequests {
```

Comment:
it seems I merged another PR with the suggested cahnges in this PR :sight: but we should do the change suggested by Will instead.